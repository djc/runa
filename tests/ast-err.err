tests/ast-err.rns [2.1]: unexpected token INDENT ('')
    pass
^
