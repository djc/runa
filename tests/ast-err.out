tests/ast-err.rns [1.11]: expected 'NL', got 'Colon'
def main()
          ^
